astropy/utils/iers/iers.py

```diff
@@ -132,6 +147,11 @@ class Conf(_config.ConfigNamespace):
     remote_timeout = _config.ConfigItem(
         10.0,
         'Remote timeout downloading IERS file data (seconds).')
+    iers_degraded_accuracy = _config.ConfigItem(
+        ['error', 'warn', 'ignore'],
```

Comment:
Well there is `metadata_conflicts` with 'error', 'warn', and 'silent', and `parse_strict` with 'raise', 'warn', and 'silent'.  I don't think there is any one answer that will get us to being "consistent". I came to 'error' and 'ignore' in part being consistent with Python warnings filter terminology.

Anyway, I don't feel too strongly since there is no right answer. @eerovaher - let me know if you still would like a change.

Reply:
It would be better to be consistent, but if the values used here are the same as the ones from Python warnings filtering then it looks like we should rather change the other values to match what is used here.

Reply:
OK, I'm just going to leave this.